common/scala/src/main/scala/org/apache/openwhisk/core/connector/Message.scala

```diff
@@ -57,7 +57,7 @@ case class ActivationMessage(override val transid: TransactionId,
                              activationId: ActivationId,
                              rootControllerIndex: ControllerInstanceId,
                              blocking: Boolean,
-                             content: Option[JsObject],
+                             content: Option[JsValue],
```

Comment:
For common action to pass parameter as dict, it would use `Option[JsObject]`
For sequence action to pass parameter as array, it would use `Option[JsArray]`